src/ByteBuffer.cpp

```diff
@@ -416,15 +458,15 @@ bool cByteBuffer::ReadVarInt32(UInt32 & a_Value)
 	CHECK_THREAD
 	CheckValid();
 	UInt32 Value = 0;
-	int Shift = 0;
-	unsigned char b = 0;
+	std::size_t Shift = 0;
+	unsigned char ReadByte = 0;
```

Comment:
I think I prefer the simple `b` here, since English has the unfortunate ambiguity in `Read` (past tense vs imperative). `ReadByte` sounds like `doReadByte` to me, rather than `theByteThatWasRead`